torch/csrc/distributed/autograd/context/dist_autograd_container.h

```diff
@@ -50,6 +68,8 @@ class DistAutogradContainer {
   DistAutogradContainer(DistAutogradContainer&&) = delete;
   DistAutogradContainer& operator=(DistAutogradContainer&&) = delete;
 
+  void checkInitialized();
+
   // Auto incrementing context id used to identify unique autograd passes.
   // Initialized with the first 16 bits being the worker_id.
   int64_t next_context_id_;
```

Comment:
Why is some places in the codebase we use camelCase for member variables and other places underscore_seperated ? What's the convention ?

Reply:
https://github.com/pytorch/pytorch/issues/26317